articles/application-insights/app-insights-export-power-bi.md

```diff
@@ -79,6 +79,17 @@ Install [Power BI Desktop](https://powerbi.microsoft.com/en-us/desktop/).
     ![Select visualization](./media/app-insights-export-power-bi/publish-power-bi.png)
 4. Refresh the report manually at intervals, or set up a scheduled refresh on the options page.
 
+## Troubleshooting
+
+### 401 or 403 Unauthorized 
+This can happen if your refesh token has not been updated. Try these steps to ensure you still have access. If you do have access and refershing the credentials does not work, please open a support ticket.
+
+1. Log into the Azure Portal and make sure you can access the resource
+2. Try to refresh the credentials for the Dashboard
+
+### 502 Bad Gateway
+This is usually caused by a query that returns too much data. You should try using a smaller time range or by using the [ago](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-analytics-reference#ago) or [startofweek/startofmonth](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-analytics-reference#startofweek) functions only [project](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-analytics-reference#project-operator) the fields you need. If reducing the amount of data does not help, please open a support ticket.
+
```

Comment:
one more option that the user has is to update the power BI to work directly with API api: see http://stackoverflow.com/questions/41869170/query-from-powerbi-to-ai-suddenly-fails-with-502-bad-gateway/43298902#43298902